pkg/cluster/private_registry_test.go

```diff
@@ -33,12 +32,13 @@ func TestGeneratePrivateRegistryDockerConfig(t *testing.T) {
 	}
```

Comment:
Can you add a comment to this test stating that it can't be run in parallel given that it modifies settings?